Create Speakers “baptistemesta”

noelmace opened the pull request:
Automatically generated by Netlify CMS

noelmace commented:
@baptistemesta I created your profile based on your bio at QCon NYC & DevoxxFR. Could you please tell me quickly if you're good with that, or make some modification? Thx

baptistemesta commented on src/pages/speakers/baptistemesta.md:
```diff
@@ -0,0 +1,31 @@
+twitter: 'https://twitter.com/baptistemesta'
+github: 'https://github.com/baptistemesta'
+linkedin: 'https://www.linkedin.com/in/baptistemesta/'
+previoustalks: >-
```
```suggestion
previoustalks: >-
* [DeveloperWeek](https://www.developerweek.com/) 2019: [Gradle: One Technology to Build Them All
](https://developerweek2019.sched.com/event/JJ2K)

```

baptistemesta commented:
LGTM 👍 :shipit:

Only added developerweek talk

noelmace commented:
Oh, snap, I didn't see that your suggestion broke the build ... I fixed it 5b0a897
Yep, I forgot to enable the outgoing webhooks in Netlify... I fixed it too :sweat_smile:
